_posts/2017-08-01-emldown.md

```diff
@@ -0,0 +1,70 @@
+
+That said, it's not always fun to write standard metadata. Although there's a very good package for helping you create the EML, [rOpenSci's EML package](https://github.com/ropensci/eml), documenting the data can be quite tedious.  Furthermore, before you share the data on a public repository that enforces EML, the only prize you get is a happy conscience, which isn't very tangible. In our unconf project, we created immediate gratification for EML users: a package that transforms the non-human readable EML file into a pretty documentation website for any dataset! 
+
+# What's EML, exactly, and why can't a human read it?
+
+[EML](https://en.wikipedia.org/wiki/Ecological_Metadata_Language) is a metadata standard originally created for the ecological sciences. In practice it's a set of XML schema documents, telling you what you need to document (e.g. the dataset creator, geographic coverage of the data, etc.) and how to build those documents (format of the XML). As mentioned previously, the [`EML` R package](https://ropensci.github.io/EML/) provides tools to create and read EML without needing to learn about XML thanks to helper functions and extensive documentation. Although its name contains the word _ecological_, one can use EML for documenting other datasets.  One of our team members uses it for documenting epidemiological datasets, because everything she wants to document is present in the standard, and because of the existence of the `EML` R package.
+
+After creating EML documentation for your data, you get an XML document that's, well... not very human readable.
+
+![raw eml](/assets/blog-images/2017-08-01-emldown/screenshot_raw_xml.png)
+
+(!!! function not working currently) In the `EML` package there's a function called `eml_view` relying on the [`listviewer` package](https://github.com/timelyportfolio/listviewer) to produce an interactive view of the XML in the Viewer pane of say, RStudio, which allows one to check some things quickly but which is far from being an user-friendly representation of the metadata.  Our goal with `emldown` was to bridge this gap and provide a quick and easy way to take EML and convert it to a more user-friendly web page.
+
+# What does `emldown` do with your EML?
+
+After installing the package, currently from Github, you can apply it to your EML...
+
+```r
+devtools::install_github("ropenscilabs/emldown")
+library("emldown")
+render_eml(path_to_my_eml)
+```
+
+and get something like this!
+
+![emldown](/assets/blog-images/2017-08-01-emldown/screenshot_emldown.png)
+
+This format is much more likely to make you and your collaborators happy. Note how little effort you needed to invest into making it! Viewing your metadata in this way makes it easy to read, and easy to spot any errors.
```

Comment:
_This format is much more likely to make you and your collaborators happy._

Maybe say something like the format is more engaging, visual, easy to digest etc?